flaky test: `chainer_tests/functions_tests/loss_tests/test_squared_error.py::TestSquaredError`

https://jenkins.preferred.jp/job/chainer/job/chainer_pr/1924/TEST=chainer-py2,label=mn1-p100/console

Occurred in #7981

>`FAIL tests/chainer_tests/functions_tests/loss_tests/test_squared_error.py::TestSquaredError_param_1_{dtype=float16, shape=(4, 3, 2)}::test_double_backward_cpu`

```
20:28:48 _ TestSquaredError_param_1_{dtype=float16, shape=(4, 3, 2)}.test_double_backward_cpu _

:

20:28:48 E           AssertionError: Parameterized test failed.
20:28:48 E           
20:28:48 E           Base test method: TestSquaredError.test_double_backward_cpu
20:28:48 E           Test parameters:
20:28:48 E             dtype: <type 'numpy.float16'>
20:28:48 E             shape: (4, 3, 2)
20:28:48 E           
20:28:48 E           
20:28:48 E           (caused by)
20:28:48 E           AssertionError: check_double_backward failed (eps=0.01 atol=0.4 rtol=0.4)

:

20:28:48 E           gradients (numeric):  -0.445741321892
20:28:48 E           gradients (backward): 0.00480155698441
20:28:48 E           
20:28:48 E           
20:28:48 E           Not equal to tolerance rtol=0.4, atol=0.4
20:28:48 E           
20:28:48 E           (mismatch 100.0%)
20:28:48 E            x: array(-0.44574132189154625)
20:28:48 E            y: array(0.004801556984411981)
20:28:48 E           
20:28:48 E           assert_allclose failed: 
20:28:48 E             shape: () ()
20:28:48 E             dtype: float64 float64
20:28:48 E             i: (0,)
20:28:48 E             x[i]: -0.445741321892
20:28:48 E             y[i]: 0.00480155698441
20:28:48 E             relative error[i]: 93.8326631005
20:28:48 E             absolute error[i]: 0.450542878876
20:28:48 E           x: -0.44574132189154625
20:28:48 E           y: 0.004801556984411981
```

--- a/tests/chainer_tests/functions_tests/loss_tests/test_squared_error.py
+++ b/tests/chainer_tests/functions_tests/loss_tests/test_squared_error.py
@@ -58,6 +58,7 @@ class TestSquaredError(unittest.TestCase):
         gradient_check.check_backward(
             functions.squared_error,
             (x0_data, x1_data), y_grad, eps=1e-2,
+            dtype=numpy.float64,
             **self.check_backward_options)
 
     def test_backward_cpu(self):
@@ -73,6 +74,7 @@ class TestSquaredError(unittest.TestCase):
         gradient_check.check_double_backward(
             functions.squared_error,
             (x0_data, x1_data), y_grad, (ggx0_data, ggx1_data), eps=1e-2,
+            dtype=numpy.float64,
             **self.check_double_backward_options)
 
     def test_double_backward_cpu(self):

